<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NS控制台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCameraFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>采集设备</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>视频:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCameraList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>240</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCapture">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>23</y>
       <width>60</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>截图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRescan">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>25</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>重新检索</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>25</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>FPS:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxFps">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>25</y>
       <width>60</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>音频:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAudioList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>240</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkMute">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>静音</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>465</y>
      <width>641</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>控制设备</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>设备:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxSerialList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>240</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSerialRescan">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>重新检索</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>641</width>
      <height>222</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>实时控制</string>
     </attribute>
     <widget class="QLabel" name="label_d">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>d</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_e">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>35</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>L e</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_g">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>53</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>C g</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_c">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>110</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>左 c</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_f">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>85</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>上 f</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rl">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>,</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_b">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>110</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>右 b</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_q">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>8</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>ZL q</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_v">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>135</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>下 v</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_w">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>35</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>w</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_l">
      <property name="geometry">
       <rect>
        <x>465</x>
        <y>60</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>A l</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_r">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>28</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>- r</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_i">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>35</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>X i</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_k">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>85</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>B k</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_o">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>8</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>ZR o</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_x">
      <property name="geometry">
       <rect>
        <x>158</x>
        <y>63</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_a">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rb">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>135</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rr">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rc">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>113</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>'</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rt">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>85</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_y">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>28</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>+ y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_u">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>35</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>R u</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_h">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>53</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>H h</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_j">
      <property name="geometry">
       <rect>
        <x>415</x>
        <y>60</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>Y j</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_s">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>85</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 170)</string>
      </property>
      <property name="text">
       <string>s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_action">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>521</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实时命令：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>脚本控制</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCOM1">
   <property name="text">
    <string>COM1</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionCamera1">
   <property name="text">
    <string>Camera1</string>
   </property>
  </action>
  <action name="actionCamera2">
   <property name="text">
    <string>Camera2</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRescan">
   <property name="text">
    <string>Rescan</string>
   </property>
  </action>
  <action name="actionAudio1">
   <property name="text">
    <string>Audio1</string>
   </property>
  </action>
  <action name="actionAudio2">
   <property name="text">
    <string>Audio2</string>
   </property>
  </action>
  <action name="actionRescan_2">
   <property name="text">
    <string>Rescan</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
